README.md

```diff
@@ -1,40 +1,12 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
 
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="mylogo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
```

Comment:
- [ ] The image of the logo is broken, Kindly remove it or use your logo in place of it if you have one. 👍 

![image](https://user-images.githubusercontent.com/77106962/211049762-a35d3c2a-049f-452c-8a00-3fd177185b8b.png)